Compiler/Saltarelle.Compiler.Tests/CompilerTests/MethodCompilation/Statements/SwitchTests.cs

```diff
@@ -92,6 +92,29 @@ public class SwitchTests : MethodCompilerTestBase {
 ");
 		}
 
+        [Test]
+        public void CanSwitchOnChar()
+        {
+            AssertCorrect(
+@"public void M() {
+	char s = 'x';
+	// BEGIN
+	switch (s) {
+		case 'X':
+			int x = 0;
+			break;
+	}
+	// END
+}",
+@"	switch ($s) {
+		case 'X': {
+			var $x = 0;
+			break;
+		}
+	}
+");
+        }
+
 		[Test]
```

Comment:
chars are treated as numeric values, so the proper translation is ```case 88:``` (unicode value of 'X').